deprecated-dune-v1-abstractions/ethereum/dex/trades/insert_1inch.sql

```diff
@@ -318,6 +318,70 @@ WITH rows AS (
 
         UNION ALL
 
+        -- 1inch Unoswap for V5 Router
+        SELECT
+            call_block_time as block_time,
+            '1inch' AS project,
+            'UNI v2' AS version,
+            'Aggregator' AS category,
+            t."from" AS trader_a,
```

Comment:
the traces table using alias `t` is not accessible in this outer statement. But `trader_a` is already defined on line 360, so we can probably just reference that here?